src/components/Contacts/ContactDetails/ContactDetailsTab/ContactDetailsTab.tsx

```diff
@@ -36,11 +36,13 @@ const ContactDetailHeadingText = styled(Typography)(() => ({
   fontWeight: 'bold',
 }));
 
-const ContactDetailLoadingPlaceHolder = styled(Skeleton)(({ theme }) => ({
-  width: '100%',
-  height: '24px',
-  margin: theme.spacing(2, 0),
-}));
+export const ContactDetailLoadingPlaceHolder = styled(Skeleton)(
```

Comment:
I don't think it affects bundle sizes in this case, but I still prefer to move exported styled components into a separate file.